stingray/powerspectrum.py

```diff
@@ -1064,6 +1071,57 @@ def _make_matrix(self, lc):
         self.time = tstart + 0.5 * self.segment_size
         self.df = avg.df
         self.dt = self.segment_size
+        self.meanrate = avg.nphots / avg.n / avg.dt
+        self.nphots = avg.nphots
+        self.m = 1
+
+    def power_colors(
+        self,
+        frequency_edges=[1 / 256, 1 / 32, 0.25, 2.0, 16.0],
+        frequencies_to_exclude=None,
+        poisson_power=None,
+    ):
+        """
+        Return the power colors of the dynamical power spectrum.
```

Comment:
I would say "compute" rather than "return" here, but that's a minor thing. Should also have citation to Heil paper, maybe?

Reply:
Done